common/BUILD

```diff
@@ -41,28 +35,17 @@ java_library(
     tags = ["maven_coordinates=grakn.core:common:{pom_version}"],
 )
 
-genrule(
-    name = "grakn-version",
-    srcs = [
-        "templates/util/GraknVersion.java",
-        "//:VERSION",
-    ],
-    cmd = "VERSION=`cat $(location //:VERSION)`;sed -e \"s/VERSION_PLACEHOLDER/$$VERSION/g\" $(location templates/util/GraknVersion.java) >> $@",
-    outs = ["util/GraknVersion.java"]
-)
 
 deploy_maven_jar(
     name = "deploy-maven-jar",
-    targets = [":common"],
+    target = ":common",
+    package = "common",
+    deps = [
+        "//common/exception",
+        "//common/config",
+        "//common/http",
+        "//common/util"
+    ],
```

Comment:
Let's move the `deps` to be declared in the originating `target`, and have `deploy_maven_jar` to traverse the dependency tree and build the jar. One idea is to use an `aspect`, but we can look into other ways too.

Reply:
@haikalpribadi solved it by having `jarjar_library` which takes a number of `java_library` targets and produces one JAR out of them